index.html

```diff
@@ -42,5 +44,134 @@ <h2>LET'S CONNECT</h2>
+            </div>
+            <ul class="prog-lang">
+                <li class="lang">html</li>
+                <li class="lang">css</li>
+                <li class="lang">javascript</li>
+            </ul>
+            <button class="project-link" type="button">See Project</button>
+        </div>
+    </section>
+    <section class="about-me">
+        <div class="about-container">
+            <h1 class="about-title">About Myself</h1>
+            <div class="about-body">
+                <p>I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            </div>
+            <div class="connect">
+                <h2>LET'S CONNECT</h2>
+            </div>
+            <ul class="social-media-icons">
+                <li class="social-media-icon"><a href="https://twitter.com/ebidien83"><img src="twitter.png"
+                            alt="twitter" /></a></li>
+                <li class="social-media-icon"><a href="https://www.linkedin.com/in/ebidien"><img src="linkedin.png"
+                            alt="linkedin" /></a></li>
+                <li class="social-media-icon"><a href="https://medium.com/@ebidien"><img src="medium.png"
+                            alt="medium" /></a></li>
+                <li class="social-media-icon"><a href="https://github.com/ebidien"><img src="github.png"
+                            alt="github" /></a></li>
+                <li class="social-media-icon"><a href="https://angel.co/u/dienebi-ombu"><img src="angellist.png"
+                            alt="angellist" /></a></li>
+            </ul>
+            <button class="resume-link" type="button">Get my resume</button>
+            <section class="about-stack">
+                <div class="stack-container">
+                    <div class="stack-category">
+                        <h2>Languages</h2>
+                        <i class="fa fa-chevron-down"></i>
+                    </div>
+                    <ul class="category-list">
+                        <li class="category-item"><img class="stack-icon" src="js-icon.png" alt=""><h3 class="stack-item">JavaScript</h3></li>
+                        <li class="category-item"><img class="stack-icon" src="html-icon.png" alt=""><h3 class="stack-item">HTML</h3></li>
+                        <li class="category-item"><img class="stack-icon" src="css-icoon.png" alt=""><h3 class="stack-item">CSS</h3></li>
+                    </ul>
+                </div>
+                <div class="stack-category">
+                    <h2>Frameworks</h2>
+                    <i class="fa fa-chevron-right"></i>
+                </div>
+                <div class="stack-separator"></div>
+                <div class="stack-category">
+                    <h2>Skills</h2>
+                    <i class="fa fa-chevron-right"></i>
```

Comment:
- [ ] Please use this color `#ACB7C3` for the drop-down menu to match the design

Reply:
Corrected the color inline thus: `<i class="fa fa-chevron-right" style="color: acb7c3;"></i>`